nautobot/utilities/drf_views.py

```diff
@@ -422,6 +399,7 @@ def view(request, *args, **kwargs):
             self.request = request
             self.args = args
             self.kwargs = kwargs
+            self.action = action
```

Comment:
Is this the same `self.action` as the one coming in from DRF views? Up there it appears there might be a leak of `action` variable from iterating over `actions.items()`?

I'm concerned that this is getting set here because otherwise this method appears to be not much different than the default method!